packages/core/src/credentials-provider/DefaultCredentialsProvider.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { CredentialProvider } from './CredentialsProvider';
+import { EnvCredentialProvider } from './DefaultCredentialProvider';
+
+describe('EnvCredentialProvider', () => {
+  it('sholud create new instance', () => {
+    const provider = new EnvCredentialProvider();
+    expect(provider).toBeDefined();
+  });
+
+  describe('get', () => {
+    const OLD_ENV = process.env;
+    let provider: CredentialProvider;
+
+    beforeAll(() => {
+      provider = new EnvCredentialProvider();
+    });
+
+    beforeEach(() => {
+      jest.resetModules();
+      process.env = { ...OLD_ENV };
```

Comment:
Mutating objects is not acceptable, you should use `spy` https://github.com/NeuraLegion/secbox-sdk-js/pull/1#discussion_r832892001